include/open62541/client_highlevel.h

```diff
@@ -656,9 +656,21 @@ typedef UA_StatusCode (*UA_NodeIteratorCallback)(UA_NodeId childId, UA_Boolean i
                                                  UA_NodeId referenceTypeId, void *handle);
 #endif
 
+typedef struct
+{
+    UA_BrowseDirection direction;
+    bool includeSubtypes;
```

Comment:
Use the UA_Boolean type.

The opening brace can be moved up one line in our overall coding style.

Reply:
![image](https://github.com/open62541/open62541/assets/86729310/4cb14aa7-3736-43c1-86b9-30f47988a765)

Done. :white_check_mark: